graderutils_format/templates/feedback.html

```diff
@@ -42,12 +42,16 @@ <h3 class="testgroup-header">{{ result_group.title }}</h3>
         <div class="panel-heading">
             <h5>
                 <a data-toggle="collapse"
-                   href="#result{{ result_group_loop.index }}-{{ test_result_loop.index }}">{{ result.title }}</a>
+                    href="#result{{ result_group_loop.index }}-{{ test_result_loop.index }}">{{ result.title }}</a>
```

Comment:
These `id`s should be prefixed with a random number at some point. A+ can have multiple feedbacks visible at the same time, thus these will eventually collide. Let's leave that for later, just letting you know.